src/subdomains/core/statistic/cfp.service.ts

```diff
@@ -161,10 +167,21 @@ export class CfpService implements OnModuleInit {
     return cfpList.reverse();
   }
 
+  async getCurrentProposals(): Promise<Proposal[]> {
+    const currentProposals = await this.client.listProposal();
+    this.masternodeCount = await this.client.getProposal(currentProposals[0].proposalId).then((p) => p.votesPossible);
+
+    for (const proposal of currentProposals) {
+      proposal.proposalVotes = await this.getVotes(await this.client.listVotes(proposal.proposalId));
+    }
```

Comment:
I would move this to the `getCfpResult` method and remove the `proposalVotes` field on `Proposal`. It's misleading because one could think that the votes are returned with the `listProposal`.